examples/events.rs

```diff
@@ -32,7 +32,9 @@ async fn run() {
         .instance
         .event::<_, (Address, Address, U256)>("Transfer")
         .expect("transfer event not found")
-        .stream();
+        .topic0(Topic::This(accounts[0])) // from
```

Comment:
Not the nicest API, but I assume we will have to wait until we do code-generation to have named arguments here...

Reply:
Exactly, that's what I'm working on now.

Reply:
@e00E also suggested `all_events` return an `enum Event { Transfer(...), Mint(...), etc. }`.